doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -117,7 +117,10 @@ For lists of protocol-specific changes, see `Protocols`_.
 Bluetooth samples
 -----------------
 
-|no_changes_yet_note|
+* :ref:`ble_nrf_dm` sample:
+
+  * Split the configuration of the Distance Measurement module from the Nordic Distance Measurement library.
+    This allows to use the Nordic Distance Measurement library without the Distance Measurement module.
```

Comment:
```suggestion
    This allows the use of the Nordic Distance Measurement library without the module.
```
We might want to avoid repetition and just talk about the module (without Distance Measurement) in the second sentence. Also you could potentially drop also the name of the library, as it is mentioned in the previous sentence.

Reply:
Thanks, fixed.